packages/strapi-plugin-upload/config/functions/bootstrap.js

```diff
@@ -31,11 +28,37 @@ module.exports = async () => {
   registerPermissionActions();
 };
 
+const wrapFunctionForErrors = fn => async (...args) => {
+  try {
+    return await fn(...args);
+  } catch (err) {
```

Comment:
In term of design here is what I think would be nice to have:


```js
try {

} catch (err) {
 if (errors.is(err, errors.ENTITY_TOO_LARGE)) {
  throw entityTooLarge
 } else {
 throw badImplementation
}
}

```


using this allow to encapsultabe the checking logic (e.g err.type)